packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -268,15 +266,24 @@ export function getWebpackConfig(bundle: Bundle, worker: WorkerConfig) {
         {
           test: /\.(js|tsx?)$/,
           exclude: /node_modules/,
-          use: {
-            loader: 'babel-loader',
-            options: {
-              babelrc: false,
-              presets: IS_CODE_COVERAGE
-                ? [ISTANBUL_PRESET_PATH, BABEL_PRESET_PATH]
-                : [BABEL_PRESET_PATH],
+          use: [
+            {
+              loader: require.resolve('./public_path_loader.js'),
+              options: {
+                test: bundle.entry,
```

Comment:
I think this `test` options is sort of reimplementing loader matching, we should instead just apply this loader before `babel-loader` and only apply it to the files that we care about, maybe with something like:

```ts
{
  // ...
  module: {
    rules: [
      {
        include: bundle.entry,
        loader: require.resolve('./public_path_loader.js'),
        options: {
          key: bundle.id,
        },
      },
      /* ... original babel-loader rule ... */
    ]
  }
}
```